DnD-Montreal/session-tome
# Fix entry fields saving / displaying improperly

## Description
Closes #326 

Formerly when editing or saving an entry (DM or character) the fields Length, GP, and Levels would always display 0. Additionally length would not be saved in the database.

This Fixes those issues by properly targeting the passed prop data, and adding missing validation rules for Length

## How to Test
1. Login to a user with Characters / Entries
2. Edit the Length / Levels / GP
3. Make sure the values shown are correct
